helm_deploy/values-dev.yaml

```diff
@@ -1,11 +1,11 @@
 ---
-# Per environment values which override defaults in hmpps-template-kotlin/values.yaml
+# Per environment values which override defaults in prisoner-transactions-api/values.yaml
 
 generic-service:
   replicaCount: 2
 
```

Comment:
If you want to switch off the ingress IP checks (so its open to any) add this like under the generic service label:
`allowlist: null`

Reply:
Just for DEV though.

Reply:
Ignore me - you don't want the API to be accessible outside!  I meant this in the front-end service.